library/examples/hate_speech.py

```diff
@@ -36,7 +36,7 @@
     pretrained_model="distilbert-base-uncased",
     user_name="semy",
     repo_name="finetuning-tweeteval-hate-speech",
-    save_remote=False,
+    save_remote=True,
```

Comment:
do you want to set this to True? I don't mind

Reply:
I was testing the new login and had to set it True -- this is actually why we should have a global setting so that I can tweak it without needing to commit (this setting is hidden in the file structure).